api/src/main/data/bt_language_names.xml

```diff
@@ -742,6 +798,58 @@ QUESTIONS?
 <symbol>TRADITIONAL_CHINESE</symbol>
 </language>
 <language>
+<id>90</id>
+<name>Mandarin Chinese</name>
+<doc>Mandarin Chinese</doc>
+<iso639-3>cmn</iso639-3>
+<iso639-1>zh</iso639-1>
+<basis_iso639_1_suffix>_mc</basis_iso639_1_suffix>
```

Comment:
I agree with Joel [in COMN-225](https://basistech.atlassian.net/browse/COMN-225?focusedCommentId=231257) that we should avoid suffixes.